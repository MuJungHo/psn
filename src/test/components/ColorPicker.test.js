import React from 'react'
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import ColorPicker from '../../components/ColorPicker'

configure({ adapter: new Adapter() });

describe('ColorPicker', () => {
  const colors = ['#4A90E2', '#4ACAAD', '#9759CD', '#F6B03C', '#EE6778', '#4A4A4A']
  const component = shallow(<ColorPicker colors={colors} />);
  it('should match the snapshot', () => {
    expect(component.html()).toMatchSnapshot();
  });
})