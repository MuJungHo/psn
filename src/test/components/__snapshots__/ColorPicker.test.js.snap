// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorPicker should match the snapshot 1`] = `"<div class=\\"makeStyles-colorGroup-1\\"><span class=\\"MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-4 MuiRadio-root Component-root-2 MuiRadio-colorSecondary MuiIconButton-colorSecondary\\" aria-disabled=\\"false\\"><span class=\\"MuiIconButton-label\\"><input type=\\"radio\\" class=\\"PrivateSwitchBase-input-7\\" value=\\"#4A90E2\\"/><div class=\\"PrivateRadioButtonIcon-root-8\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\\"></path></svg><svg class=\\"MuiSvgIcon-root PrivateRadioButtonIcon-layer-9\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z\\"></path></svg></div></span></span><span class=\\"MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-4 MuiRadio-root Component-root-11 MuiRadio-colorSecondary MuiIconButton-colorSecondary\\" aria-disabled=\\"false\\"><span class=\\"MuiIconButton-label\\"><input type=\\"radio\\" class=\\"PrivateSwitchBase-input-7\\" value=\\"#4ACAAD\\"/><div class=\\"PrivateRadioButtonIcon-root-8\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\\"></path></svg><svg class=\\"MuiSvgIcon-root PrivateRadioButtonIcon-layer-9\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z\\"></path></svg></div></span></span><span class=\\"MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-4 MuiRadio-root Component-root-13 MuiRadio-colorSecondary MuiIconButton-colorSecondary\\" aria-disabled=\\"false\\"><span class=\\"MuiIconButton-label\\"><input type=\\"radio\\" class=\\"PrivateSwitchBase-input-7\\" value=\\"#9759CD\\"/><div class=\\"PrivateRadioButtonIcon-root-8\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\\"></path></svg><svg class=\\"MuiSvgIcon-root PrivateRadioButtonIcon-layer-9\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z\\"></path></svg></div></span></span><span class=\\"MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-4 MuiRadio-root Component-root-15 MuiRadio-colorSecondary MuiIconButton-colorSecondary\\" aria-disabled=\\"false\\"><span class=\\"MuiIconButton-label\\"><input type=\\"radio\\" class=\\"PrivateSwitchBase-input-7\\" value=\\"#F6B03C\\"/><div class=\\"PrivateRadioButtonIcon-root-8\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\\"></path></svg><svg class=\\"MuiSvgIcon-root PrivateRadioButtonIcon-layer-9\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z\\"></path></svg></div></span></span><span class=\\"MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-4 MuiRadio-root Component-root-17 MuiRadio-colorSecondary MuiIconButton-colorSecondary\\" aria-disabled=\\"false\\"><span class=\\"MuiIconButton-label\\"><input type=\\"radio\\" class=\\"PrivateSwitchBase-input-7\\" value=\\"#EE6778\\"/><div class=\\"PrivateRadioButtonIcon-root-8\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\\"></path></svg><svg class=\\"MuiSvgIcon-root PrivateRadioButtonIcon-layer-9\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z\\"></path></svg></div></span></span><span class=\\"MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-4 MuiRadio-root Component-root-19 MuiRadio-colorSecondary MuiIconButton-colorSecondary\\" aria-disabled=\\"false\\"><span class=\\"MuiIconButton-label\\"><input type=\\"radio\\" class=\\"PrivateSwitchBase-input-7\\" value=\\"#4A4A4A\\"/><div class=\\"PrivateRadioButtonIcon-root-8\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\\"></path></svg><svg class=\\"MuiSvgIcon-root PrivateRadioButtonIcon-layer-9\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z\\"></path></svg></div></span></span></div>"`;
